src/css/custom.css

```diff
@@ -225,4 +225,12 @@ table, tr, td, th {
 
 .hard {
   color: rgb(255 45 85);
+}
+
+/*  for emoji's */
+
+@media screen and (min-width: 1000px) and (max-width: 1119px) {
```

Comment:
mobile first principles please. no max-width.

Reply:
```css
  @media screen and (min-width: 997px) and (max-width: 1119px) {
    .nav-emoji{
       display: none;
    }
  }
  ```

Reply:
I used for between size (997px - 1119px) only not for other

Reply:
```css
@media screen and (min-width: 997px) {
  .nav-emoji{
    display: none;
  }
}
@media screen and (min-width: 1120px) {
  .nav-emoji{
    display: inline-block;
  }
}
```